File: cypress/cypress/pages/securePO.js
class securePO{
    elements ={
        SecurePage_Title_text : () => cy.xpath("//h1[text()='Secure Area']"),     
    }
    Verify_Expected_Elements_Appear(){
        this.elements.SecurePage_Title_text().should('be.visible')
    }   
}

module.exports = new securePO();

require('cypress-xpath')

File: capybara/spec/pages/securePO.rb
# spec/securePO.rb
class SecurePO
    include Capybara::DSL
    include RSpec::Matchers
  
    def initialize
       @SecurePage_Title_text  = '//h1[text()=\'Secure Area\']' 
    end

    def Verify_Expected_Elements_Appear
      expect(find(:xpath, @SecurePage_Title_text ).visible?).to be(true)
    end
  end